.vs/config/applicationhost.config

```diff
@@ -170,7 +170,7 @@
             </site>
             <site name="P202.Training.WCF" id="3">
                 <application path="/" applicationPool="Clr4IntegratedAppPool">
-                    <virtualDirectory path="/" physicalPath="D:\dev\P202-Training\P202.Training.WCF" />
+                    <virtualDirectory path="/" physicalPath="C:\GitHub\P202-Training\P202.Training.WCF" />
```

Comment:
I think it would be better if we add this file to the gitignore since it will have a different path for each of us (my bad, I should have done that when creating the initial structure). Would you mind to fix that?